packages/hash/frontend/src/components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolUpdateEntity.ts

```diff
@@ -111,6 +111,10 @@ export const useBlockProtocolUpdateEntity = (
           query: getEntityTypeRootedSubgraphQuery,
```

Comment:
We should get rid of this line in the query calls

Reply:
I will do that when I touch the entity endpoints